amlnx_adapter/amlnx_adapter/ArangoDB_NetworkxAdapter.ipynb

```diff
@@ -0,0 +1,88 @@
+{
+ "nbformat": 4,
+ "nbformat_minor": 0,
+ "metadata": {
+  "language_info": {
+   "name": "python",
+   "pygments_lexer": "ipython3"
+  }
+ },
+ "cells": [
+  {
+   "cell_type": "code",
+   "metadata": {},
+   "source": [
+    "!pip install amlnx-adapter\n",
+    "!pip install python-arango\n",
+    "!pip install PyYAML==5.1.1"
+   ],
+   "execution_count": null,
+   "outputs": []
+  },
+  {
+   "cell_type": "code",
+   "metadata": {},
+   "source": [
+    "from amlnx_adapter.graph_params import GraphParams\n",
+    "from amlnx_adapter.arangoDB_networkx_arango_adapter import ArangoDB_Networkx_Adapter\n",
+    "gp = GraphParams()\n",
+    "cfg = {}\n",
+    "cfg[gp.DB_USER_NAME] = \"TUThteu92kx6d6vy5bv8hjdgi\"\n",
+    "cfg[gp.DB_NAME] = \"TUTunnln6vscrstef2r6mz5b\"\n",
+    "cfg[gp.DB_CONN_PROTOCOL] = 'https'\n",
+    "cfg[gp.DB_PASSWORD] = 'TUT2p6okjbkwd5ygusspxc9gl'\n",
+    "cfg[gp.DB_SERVICE_HOST] = '5904e8d8a65f.arangodb.cloud'\n",
+    "cfg[gp.DB_VERTEX_COL] = {'account': {'Balance', 'account_type', 'customer_id', 'rank'},\\\n",
```

Comment:
IMO we should 
a) combine vertex and edge collections into one dictionary
b) should this dictionary nor be part of the the cfg.
The cfg would be imo connection information (hence I would rename it to conn), and then we can issue multtiple queries per connection 

E.g., 
```
ma = ArangoDB_Networkx_Adapter(conn = con) # only connection information
g1 = ma.create_networkx_graph(graph_definititon= 'FraudDetection', graph_config= dict{})
g2 = ma.create_networkx_graph(graph_query=aql, graph_config= dict{})```

Reply:
I can take care of this

Reply:
```attributes = { 'vertexCollections': {'Collection1': {('attribute1','target_name'), attribute2'}}, 'edgeCollections' : {Collection2' : {'att1','att2'}}```